ansible/roles/software/ckan/common/vars/bionic.yml

```diff
@@ -8,3 +8,4 @@ ckan_os_packages:
   - libxslt1-dev
   - python-dev
   - virtualenv
+  - libgdbm-dev
```

Comment:
Would you mind maintaining the alphabetical order of the list?

Reply:
Sure -- I didn't notice, I thought they were chronological ;)